compiler-rt/test/ubsan_minimal/TestCases/override-callback.c

```diff
@@ -9,8 +10,21 @@
 static int Result;
 
 void __ubsan_report_error(const char *kind, uintptr_t caller) {
+// -fsanitize-handler-preserve-all-regs is ignored on other architectures.
+// Prented we called to other handler on those.
+#if defined(PRESERVE) && !defined(__aarch64__) && !defined(__x86_64__)
```

Comment:
would be better to XFAIL them, in case they get a support in the future?

Reply:
maybe, but then we'd have to fork the test case. up to you

Reply:
Leaving as is. The compiler-rt testing is a mess, we build i368 by just adding `-m32`, so the `REQUIRES` or `XFAIL` don't work.